tests/Frameworks/ZendFramework/Version_1_12/application/controllers/SimpleController.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+class SimpleController extends Zend_Controller_Action
+{
+    public function indexAction()
+    {
+        // Don't auto render this action
+        $this->_helper->viewRenderer->setNoRender();
+        echo 'This is a string.';
+    }
+
+    public function viewAction()
+    {
+        // Empty
+    }
```

Comment:
Where are the methods for view rendering and exception  testing :) ?

Reply:
This one renders the view in `application/views/scripts/simple/view.phtml`. ZF1 does a lot of magic to render views. And the exception case is found in the `ErrorController.php`. :)